product_development.md

```diff
@@ -0,0 +1,118 @@
+# Product development
+
+## Documents & Artifacts
+
+### Kanban board
+
+The Kanban board is the main tool of communication between stakeholders and the different product development units.
+
+The Kanban board is owned by the product management team.
+
+Engineers are responsible for maintaining cards on the Kanban board once they have pulled them into the backlog. Stories are always assigned when they are in progress, but never while they are in the backlog.
+
+All work that is done in either product, design or engineering should be reflected on the board. Stakeholders may have tickets on the board too.
+
+### OKR initiatives
+
+Strategic initiatives developed with a department and prioritized in close coordination with an objective owner. Those initiatives could be features and also research tasks, surveys, content work and more.
+
+They are prioritized over anything else to ensure the success of our OKRs.
+
+OKR initiatives should be in line with our long-term product vision.
+
+OKR initiatives are usually tracking in a separate lane on the Kanban board.
+
+### New feature tickets
+
+New features are the result of analysis or research based user demand.
+The product and design team owns the requirements engineering and conception of new features.
```

Comment:
I actually don't know what "the requirements engineering" means.

Reply:
Google it ;) https://en.wikipedia.org/wiki/Requirements_engineering